packages/react-composites/tests/browser/call/hermetic/VideoGalleryLayout.test.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  addVideoStream,
+  buildUrlWithMockAdapter,
+  defaultMockCallAdapterState,
+  defaultMockRemoteParticipant,
+  test
+} from './fixture';
+import { expect } from '@playwright/test';
+import { dataUiId, waitForSelector, stableScreenshot, isTestProfileMobile, pageClick } from '../../common/utils';
+import { IDS } from '../../common/constants';
+
+test.describe('VideoGalleryLayout tests', async () => {
+  /* @conditional-compile-remove(pinned-participants) */
+  test.only('VideoTile contextual menu shows "Fit to frame" by default', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileMobile(testInfo));
```

Comment:
is there planned mobile snapshot tests also?

Reply:
Yes, will be adding tests for drawer menu for mobile devices separately.